.github/workflows/deploy-master.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: build image
+        run: docker build -t documentation:latest .
+      - name: copy static files locally
+        run: docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
```

Comment:
I opened https://github.com/docker/docker.github.io/pull/10331 to allow using the `--output` option, and to have the option of building without archives, or to build the archives separately (which could be useful if we want to build those "once", and only update the "current" docs for updates.